css/menu.css

```
@@ -1,21 +1,47 @@
-table {
+body,
+html {
+    box-sizing: border-box;
+    font-family: "proxima-nova", serif;
+    margin: 0;
+    padding: 0;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+body > main {
+    position: relative;
+    margin-top: 100px;
+}
+
+body > main > h2 {
+    color: var(--clr-on-nav);
+    margin-left: 20%;
+    margin-bottom: 2px;
+}
+
+body > main > table {
     position: relative;
     width: 60%;
     margin-left: 20%;
-    top: 30px;
     border-collapse: collapse;
+    background-color: white;
+    border-radius: 10px;
 }
 
-td:nth-child(2) {
-    width: 10%;
+body > main > table > tbody > tr > td:nth-child(2) {
```

Ik zou hiervoor een class gebruiken. Iets in de richting van:
```suggestion
table.menuTable tr > td:nth-child(2) {
```

Dat maakt het leesbaarder

Hetzelfde geldt voor de andere table selectors

---

gedaan